### PR TITLE
ci: limit rbac for the reconciler

### PR DESCRIPTION
**Pull request by SebastianBakala-TomTom**
solves 2nd question from #833

### CODE REVIEW

**Comment by yitsushi** [comment]
This PR will fix the helm-test workflow: https://github.com/weaveworks/tf-controller/pull/1147

**Comment by SebastianBakala-TomTom** [comment]
Sorry @yitsushi. I missed your comment.

File: charts/tf-controller/templates/rbac.yaml [hunk 1]
```diff
@@ -46,6 +46,69 @@ rules:
...
+  resources:
+  - jobs
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
```
Comment (yiannistri) [inline charts/tf-controller/templates/rbac.yaml:95]: Not sure if this is needed? I could find no Job object references in the code, could you explain why this was added?

**Review by yiannistri — COMMENTED** [review]
Tested and works fine. Unsure about `jobs` RBAC though.

**Review by yiannistri — APPROVED** [review]
Tested with "plan and manual apply" as well as "plan with automatic apply" and worked fine. Secrets and configmaps are also created and got no RBAC errors in the logs.

Thank you for the PR @SebastianBakala-TomTom :1st_place_medal:

**Pull request merged**